yatl/helpers.py

```diff
@@ -16,6 +16,40 @@
 # New HTML Helpers
 # ################################################################
 
+def _vk(k):
+    """validate atribute name of tag
+        @k: atribute name
+    """
+    ignore = ["_class",
+              "_id",
+              "_src",
+              "_style",
+              "_title",
+              "_value",
+              "_alt",
+              "_name",
+              "_content",
+              "_rel",
+              "_height",
+              "_width",
+              "_placeholder",
+              "_autocomplete",
+              "_href"]
```

Comment:
I'd rather not whitelist/blacklist tags. These things change with line. Leave it to the developer

Reply:
I put this list to increase the performance, since they are the most characteristic more then it would not go through the verification